### PR TITLE
Answer Button Arrow keys fixed

### LINKED ISSUE #15093: [BUG]: Unnecessary "next" card option in CardTemplatePreviewer
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

- Open a note created using the `Basic (optional reversed card)` notetype in the Note Editor. The note should not have a reversed card.
- Open the preview of the card.

### Expected behaviour

The forward card is seen in the preview and there are no `>` (next) or `<` (previous) options associated with the `Show/Hide answer` button.

(Screenshot from the preview of a card created using the `Basic` notetype)

### Actual behaviour

The `Show/Hide answer` button shows `>` (next) or `<` (previous) options.

On clicking the `>` button, it shows:

On the Desktop version, clicking the next button shows the next card (from the next note). It makes sense because the note editor and the browser are inseparable there.

However, in AnkiDroid, I think that it makes more sense to hide the button when there is no reverse card associated with the note.

### Debug info

```text
2.17 alpha 14
```

### (Optional) Anything else you want to share?

_No response_

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by soCallmeAdityaKumar**
## Purpose / Description
Unnecessary dummy card was shown on next card if reverse was empty

## Fixes
* Fixes #15093

## How Has This Been Tested?

https://github.com/ankidroid/Anki-Android/assets/101629190/1f21b198-389b-4378-8ea7-bb58720b6d0b

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.kt [hunk 1]
```diff
@@ -240,8 +240,21 @@ open class CardTemplatePreviewer : AbstractFlashcardViewer() {
             val toPreview = setCurrentCardFromNoteEditorBundle(col)
             if (toPreview != null) {
                 cardCount = toPreview.note().numberOfCardsEphemeral()
+
                 if (cardCount >= 2) {
-                    previewLayout!!.showNavigationButtons()
+                    val fields = getBundleEditFields(mNoteEditorBundle!!)
+                    if (fields.size > 2 && fields[2].isEmpty()) {
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.kt:246]: What if I have 10 fields, and the 4th is empty? Or any other combination that is not the second field being empty

**Review by BrayanDSO — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.kt (hunk 1 above)
Comment (BrayanDSO) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.kt:246]: Yep. This should match the desktop behavior as much as possible

**Comment by david-allison** [comment]
CI is failing

**Comment by BrayanDSO** [comment]
CI still fails

**Comment by BrayanDSO** [comment]
After discussion in the original issue, it was agreed that this may not be necessary.

Sorry for the time spent, but thanks for the work

**Pull request closed without merging**